Build error -  ‘std::initializer_list’ has not been declared

cs_rcu_list.h file compiles with an error:

`cs_rcu_list.h:98:43: error: ‘std::initializer_list’ has not been declared` 

Compiler: gcc (GCC) 13.1.1 20230714
on Arch Linux

--- a/src/cs_rcu_list.h
+++ b/src/cs_rcu_list.h
@@ -22,6 +22,7 @@
 
 #include <atomic>
 #include <cstddef>
+#include <initializer_list>
 #include <memory>
 #include <mutex>
 

